src/components/project/project.edgedb.repository.ts

```diff
@@ -81,9 +86,10 @@ export class ProjectEdgeDBRepository
     { filter: input }: ProjectListInput,
   ) {
     return [
-      input.type != null &&
-        // https://github.com/edgedb/edgedb-js/issues/615
-        e.op(project.__type__.name, '=', `default::${input.type}Project`),
+      // TODO: add back after edgedb piece is fixed
+      // input.type != null &&
+      // https://github.com/edgedb/edgedb-js/issues/615
+      // e.op(project.__type__.name, '=', `default::${input.type}Project`),
```

Comment:
This still needs to be addressed after edgedb change is in place.
Writing this as a reminder for myself.